packages/calcite-components/src/components/stepper/stepper.scss

```diff
@@ -1,3 +1,15 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-stepper-bar-gap: When `horizontal-single`, specifies the space between items in the step bar.
+ * @prop --calcite-stepper-bar-inactive-fill-color: When `horizontal-single`, specifies the fill color of items in the step bar.
```

Comment:
QQ: i think these tokens also applies for `horizontal` & `vertical` layouts. Though we use border in those layout as step-bar, visually they look the same. It can be consistent if users can customize step-bar color in only `horizontal-single` layout. Thoughts @ashetland

Reply:
Oh good point. Yes, if we can, I think these tokens should apply in all layouts.